app/models/reviewable.rb

```diff
@@ -534,22 +535,60 @@ def self.list_for(
     result
   end
 
+  def self.recent_list_with_pending_first(user)
```

Comment:
```suggestion
  def self.recent_list_with_pending_first(user, limit: 30)
```

Minor suggestion but I feel like method definitions like this makes it clearer that there is a limit by default.